plugins/scanners/scancode/src/main/kotlin/ScanCodeConfig.kt

```diff
@@ -19,19 +19,47 @@
 
 package org.ossreviewtoolkit.plugins.scanners.scancode
 
+import kotlin.math.max
+import kotlin.time.Duration.Companion.seconds
+
 import org.ossreviewtoolkit.utils.common.Options
+import org.ossreviewtoolkit.utils.common.splitOnWhitespace
 
 data class ScanCodeConfig(
-    val commandLine: String?,
-    val commandLineNonConfig: String?
+    val commandLine: List<String>,
+    val commandLineNonConfig: List<String>
 ) {
     companion object {
-        val EMPTY = ScanCodeConfig(null, null)
+        /**
+         * The default time after which scanning a file is aborted.
+         */
+        private val DEFAULT_TIMEOUT = 300.seconds
+
+        /**
+         * The default list of command line options that might have an impact on the scan results.
+         */
+        private val DEFAULT_COMMAND_LINE_OPTIONS = listOf(
+            "--copyright",
+            "--license",
+            "--info",
+            "--strip-root",
```

Comment:
out of scope: Shall we move `--strip-root` to `DEFAULT_COMMAND_LINE_NON_CONFIG_OPTIONS` as it does not have impact on the `ScanSummary` ?

Reply:
I agree, done in a new commit.